### ISSUE TITLE
Error when the gltf file is updated, but another scene in Godot is focused

### ISSUE THREAD

**Issue by Lamoot** [bikemurt/blender-godot-pipeline#6]
The following screenshot shows an error I get when I re-export the .gltf file from Blender and the Godot addon tries to update the related nodes in their scene but can't find them.

- _level_01_demo_ is the scene where I'm building the level. This is where I placed the _level_01_demo.gltf_ file.
- When I re-exported the file from Blender I had another scene focused in Godot, the _player_ character. The addon tries to update the nodes derived from the exported .gltf but it of course can't find them as they are in another scene.

![update_but_on_a_different_scene](https://github.com/user-attachments/assets/11da61ef-a534-46ad-beba-d4190d5dc6e3)

Ideally the addon in Godot would handle such cases and update the relevant scene. However it's likely tricky to get that working so at least a more graceful and informative fail would be enough.

**Comment by bikemurt** [comment]
Yep, unfortunately that is expected. The scene that contains the imported node must be open during the import process. I have spent time in the past trying to resolve this and couldn't (reason given below).

This was behaviour I did not see in Godot 4.2.1 stable, so somewhere between there are 4.3, the import pipeline has changed, and if there are nodes which are linked to a GLTF file, then the scene that node is in should be open when the post import script is trigger (either manually via re-importing or by exporting a .gltf file).

There might be a solution if this addon were a Godot plugin proper, because then we would have some ability to check which scene is open in the editor. Since this is just a GDscript attached to a Node3D, I don't believe there is a way to determine which scene is open. @tool scripts also would not have that level of access.

I won't close the issue in case anyone else has any ideas.
